File: Prog52aLangs/Prog52a.swift

print("Enter length: ")
var length = Int(readLine()!)! ?? 0
print("Enter width: ")
var width = Int(readLine()!)! ?? 0

var area = length * width
var perim = 2*length + 2*width
print("Area: ", area)
print("Perimeter: ", perim)

File: Prog52aLangs/Prog52a.jl
print("Enter width: ")
width = readline()
print("Enter length: ")
length = readline()

area = parse(Int64, width) * parse(Int64, length)
perimeter = 2*parse(Int64, width) + 2*parse(Int64, length)
print("Area: ", area)
print("\nPerimeter: ", perimeter, "\n")